bower.json

```diff
@@ -22,6 +22,9 @@
     "angular-uuid4": "~0.3.1",
     "angular-cache": "~4.5.0",
     "angular-route": "^1.5.0",
-    "vcard": "~0.2.3"
+    "vcard": "~0.2.3",
+    "dav": "git@github.com:irgendwie/dav.git#d33ecad1e64202ec5b35b6911708d1253064b5a7",
+    "vcard": "~0.2.3",
```

Comment:
We removed the dav library from bower in one of the last PRs.
Additionally vcard seems to be twice in this list.